lib/datasets-parquet/tlc/yellow-taxi-trip.rb

```diff
@@ -0,0 +1,112 @@
+require "parquet"
+require_relative "../dataset-parquet"
```

Comment:
We can use `Datasets::Dataset`:

```suggestion
require "datasets"
```

Reply:
fix: https://github.com/red-data-tools/red-datasets-parquet/pull/3/commits/9e12bf5e31ef4a213b71b3e9c332f29a24c41dcb#diff-c57f281cd2724a20af1c7d97a0d52c3736d21fdc605135e29d220ccecc08cbdeR5